libraries/AP_VisualOdom/AP_VisualOdom_IntelT265.cpp

```diff
@@ -62,15 +62,41 @@ void AP_VisualOdom_IntelT265::handle_vision_position_estimate(uint64_t remote_ti
     _last_update_ms = AP_HAL::millis();
 }
 
+// consume vision velocity estimate data and send to EKF, velocity in NED meters per second
+void AP_VisualOdom_IntelT265::handle_vision_speed_estimate(uint64_t remote_time_us, uint32_t time_ms, const Vector3f &vel, uint8_t reset_counter)
+{
+    const float scale_factor =  _frontend.get_pos_scale();
+    Vector3f vel_corrected = vel * scale_factor;
```

Comment:
I think while the positions from the T265 might have a scale issue, the velocity might not.
Maybe we can test the velocity alone with the scale factor as 1 until we can verify there is a correlation between the scale of the two.

Reply:
@chobitsfan, I've put a commit called, "AP_VisualOdom: scaling does not affect vision-speed-estimate" in the branch linked below that removes the scaling for velocity.  Maybe you could pull this into this PR and rebase on master?

https://github.com/ArduPilot/ardupilot/pull/14404/commits

Reply:
@rmackay9 Yes, I will do it.

Reply:
Hi @rmackay9  Do you want me to squash it after rebase?

Reply:
@chobitsfan, squashing or not squashing is fine. Unless something unexpected comes up I'll merge yours first and then rebase mine on master (which will include your PR by then).